### PR TITLE
camelCase JSON attributes in problems

### PR DESCRIPTION
**Pull request by stroem**
Depends on: https://github.com/SKF/go-rest-utility/pull/7

### CODE REVIEW

**Review by j0s — APPROVED** [review]
lgtm

**Comment by stroem** [comment]
Also added a small guard when resource is nil. 
(not really align with the title of this PR; woops)

File: authorization/policy.go [hunk 1]
```diff
@@ -44,13 +44,20 @@ func (p ActionResourcePolicy) Authorize(ctx context.Context, userID string, auth
 	}
 
 	if !ok {
+		if resource == nil {
+			return custom_problems.Unauthorized(userID, custom_problems.PolicyViolation{
```
Comment (Ekberg) [inline authorization/policy.go:48]: Nice catch!

Maybe add a small test in handler_test.go?

**Review by j0s — APPROVED** [review]

File: authorization/policy.go (hunk 1 above)
Comment (stroem) [reply authorization/policy.go:48]: Added tesst

**Review by j0s — APPROVED** [review]

**Pull request merged**